std_msgs/Float32MultiArray label_probs
sensor_msgs/PointCloud2 labelled_pc